prometheus/exporter/gpu_exporter.go

```diff
@@ -152,7 +152,7 @@ func writeMetricsToFile(index int, logDir string) {
 }
 
 func main() {
-    logDir := flag.String("gpu_log_dir", "/var/local/prom-exporters", "log written dir")   
+    logDir := flag.String("gpu_log_dir", "/datastorage/prom-exporters", "log written dir")   
```

Comment:
Should this log path on the host to persistent log data? {{ clusterinfo[ 'dataPath' ] }}/promethues

Reply:
This will override by the launch shell